scripts/generate_pip_deps_from_conda.py

```diff
@@ -0,0 +1,131 @@
+# BSD 3-Clause License
```

Comment:
this is mainly taken from pandas, so my understanding is that I need to include the bsd license